sdk/ai/azure-ai-inference/samples/sample_chat_completions_with_tracing.py

```diff
@@ -0,0 +1,191 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+DESCRIPTION:
+    This sample demonstrates how to get a chat completions response from
+    the service using a synchronous client. The sample also shows how to 
+    set default chat compoletions configuration in the client constructor,
```

Comment:
spelling (compoletions). Did I have this spelling mistake elsewhere?

Reply:
did not find elsewhere. fixed this one.